extensions/gemfire-modules-session/src/test/java/com/gemstone/gemfire/modules/session/junit/SeparateClassloaderTestRunner.java

```diff
@@ -23,7 +23,6 @@
 import java.net.URLClassLoader;
 
 /**
- * @author StackOverflow
```

Comment:
We need to investigate the origin of this code to ensure it is safe to include in the geode code base.